.github/workflows/x-keycloak-operator.yml

```
@@ -124,6 +124,30 @@ jobs:
           build-args: |
             KEYCLOAK_VERSION=${{ inputs.version }}
 
+      - name: Install Yq
+        working-directory: keycloak
+        run: sudo snap install yq
+
+      - name: Create OLM Bundle
+        working-directory: keycloak
+        run: |
+          cd operator && ./scripts/create-olm-bundle.sh ${{ inputs.version }} ${{ inputs.previous-version }} quay.io/${{ inputs.quay-org }}/keycloak-operator
+
+      - name: Compress OLM Bundle
+        working-directory: keycloak
+        run: |
+          tar --use-compress-program zstd -cf olm-bundle-keycloak-${{ inputs.version }}.tzst \
+          --exclude '*.tar.gz' \
+          -C operator/olm/${{ inputs.version }} .
+
+      - name: Upload Keycloak OLM bundle
+        uses: actions/upload-artifact@v4
+        with:
+          name: olm-bundle-keycloak
+          path: keycloak/olm-bundle-keycloak-${{ inputs.version }}.tzst
+          if-no-files-found: error
+          retention-days: 1
```

I'd increase the retention-days; if for whatever reason we have issues with releasing 1 day may result in loosing the artifact to do the manual release of the bundle.
```suggestion
          retention-days: 7
```